### PR TITLE
Added "add database columns" to form builder

### PR DESCRIPTION
**Pull request by tobias-kuendig**
This PR adds the "add database columns" feature to the form builder component. 

Let me know what you think!

magic

I also stumbled upon a bug where the datatable dropdown does not seem to work correctly when loaded in a popup. A fix is available, I'll work on a proper PR with a proof of the bug later today.

**Edit:** PR for this bugfix is here: https://github.com/octobercms/october/pull/4807

Just keep in mind to apply this fix to your installation if you are testing this PR:
https://github.com/OFFLINE-GmbH/october/commit/6ec2a85becc6f0a764013568674bde95fc19fe64

### CODE REVIEW

**Comment by w20k** [comment]
@tobias-kuendig, it looks awesome! :)

File: behaviors/IndexModelFormOperations.php [hunk 1]
```diff
@@ -127,6 +129,24 @@ public function onModelFormGetModelFields()
         ];
     }
 
+    public function onModelShowAddDatabaseColumnsPopup()
+    {
+        $columns = ModelModel::getModelColumnsAndTypes($this->getPluginCode(), Input::get('model_class'));
+        $config  = $this->makeConfig($this->getAddDatabaseColumnsDataTableConfig());
```
Comment (LukeTowers) [inline behaviors/IndexModelFormOperations.php]: Aligning `=` should only be done inside of arrays please

**Comment by LukeTowers** [comment]
@tobias-kuendig this looks pretty awesome! We'll try to get your bugfix included in 460 so that this could be supported on 460

**Comment by bennothommo** [comment]
@tobias-kuendig This looks fantastic! My only suggestion would be the terminology for "Add database columns" is probably misleading - you are not adding actual database columns, but instead adding fields based on the columns. Perhaps this could be "Add fields from database"?

**Comment by tobias-kuendig** [comment]
Good point, @bennothommo! I have changed the wording accordingly.

File: behaviors/IndexModelFormOperations.php (hunk 1 above)
Comment (tobias-kuendig) [reply behaviors/IndexModelFormOperations.php]: :ok_hand: fixed!

**Comment by tobias-kuendig** [comment]
I had to add another parameter to the `addControlToPlaceholder` method that allows to explicitly pass in a field name. Otherwise the method would only generate `text1`, `text2`, etc. 

The method now has five parameters, I hope this is still acceptable.

**Comment by tobias-kuendig** [comment]
Is there anything I can do to get this merged?

**Comment by bennothommo** [comment]
@tobias-kuendig We just need people to test it out and confirm it works with weird field types and large field sets.

**Comment by tobias-kuendig** [comment]
Is there anything specific blocking this PR? We could use this every day :grin:!

**Comment by daftspunk** [comment]
LGTM! Thanks as always @tobias-kuendig

**Comment by alxy** [comment]
Wow, this is really super cool @tobias-kuendig 
Thanks for the addition!

**Pull request merged**